core/src/main/kotlin/org/lflang/generator/ts/TSActionGenerator.kt

```diff
@@ -10,7 +10,7 @@ import java.util.*
  */
 class TSActionGenerator(
     private val actions: List<Action>,
-    private val networkMessageActions: List<String>
+    // private val networkMessageActions: List<String>
```

Comment:
Let's remember to delete all the commented-out code

Reply:
This is fixed in the commit [9acb408](https://github.com/lf-lang/lingua-franca/pull/1876/commits/9acb408821453057fa1dedd652cce0c8ddf5fa9e)